app/models/user_action_event.rb

```diff
@@ -6,4 +6,21 @@ class UserActionEvent < ApplicationRecord
   validates :details, presence: true
   validates :identifier, presence: true, uniqueness: true
   validates :event_type, presence: true
+
+  def self.setup
+    config_file_path = Rails.root.join('config', 'user_action_events.yml')
+    unless File.exist?(config_file_path)
```

Comment:
I kinda wonder if we should let it fail if the file doesn't exist. I would rather this prevent a deployment than silently failing